src/Aspire.Hosting.Python/PythonAppResourceBuilderExtensions.cs

```diff
@@ -364,18 +364,16 @@ private static IResourceBuilder<T> AddPythonAppCore<T>(
             }
         });
 
-        // Configure required environment variables for custom certificate trust when running as an executable
-        // Python defaults to using System scope to allow combining custom CAs with system CAs as there's no clean
-        // way to simply append additional certificates to default Python trust stores such as certifi.
+        // Configure required environment variables for custom certificate trust when running as an executable.
+        // TODO: Make CertificateTrustScope.System the default once we're able to validate that certificates are valid for OpenSSL. Otherwise we potentially add invalid certificates to the bundle which causes OpenSSL to error.
```

Comment:
Do we have an issue tracking this?

Reply:
https://github.com/dotnet/aspire/issues/12693